### PR TITLE
EventWizard import Ranks from FMS server

### PR DESCRIPTION
**Pull request by phil-lopreiato**
Import data using the same ajax data that Pit Display uses. Can't be tested, because I don't have FMS

### CODE REVIEW

**Comment by gregmarra** [comment]
Do you have dummy data? Can you make dummy data you think is the same
format? Will allow at least a unit test.

**Comment by phil-lopreiato** [comment]
I was thinking that, but how should I unit test javascript? We don't really have anything set up for that.

**Comment by gregmarra** [comment]
Oh good point. That could be a separate summer project.

File: static/javascript/tba_js/eventwizard_apiwrite.js [hunk 1]
```diff
@@ -326,6 +326,29 @@ $('#rankings_file').change(function(){
     reader.readAsBinaryString(f);
 });
 
+$('.update-rankings').click(function(){
+    $(this).css('background-color', '#eb9316');
+    $.ajax({
+            url: '10.0.100.5/pit/getdata?random=' + Math.random(),
+            cache: false,
+            timeout: 5000, success: function (data) {
+                rankings = ['Rank', 'Team', 'Avg', 'CP', 'AP', 'RC', 'TP', 'LP', 'Played'];
```
Comment (fangeugene) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: So we need to change this every year.

Comment (phil-lopreiato) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: Correct :/

File: static/javascript/tba_js/eventwizard_apiwrite.js [hunk 2]
```diff
@@ -326,6 +326,29 @@ $('#rankings_file').change(function(){
...
+                rankData = JSON.parse(data);
+                for(var i=0; i<rankData.length; i++){
+                    // Turn team number -> team key
+                    rankData[i]['Team'] = "frc"+rankData[i]['Team'];
+                    rankings.push(rankData[i]);
+                }
+                makeRequest('/api/trusted/v1/event/' + $('#event_key').val() + '/rankings/update', JSON.stringify(rankings), $(this));
```
Comment (fangeugene) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: Is this the right format to make the trusted API call? https://github.com/the-blue-alliance/the-blue-alliance/blob/master/datafeeds/parsers/json/json_rankings_parser.py

Comment (phil-lopreiato) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: Oh, wait, no it isn't. Will fix tonight

File: static/javascript/tba_js/eventwizard_apiwrite.js [hunk 3]
```diff
@@ -326,6 +326,29 @@ $('#rankings_file').change(function(){
...
+            },
+            error: function (error) {
+                // We had an error getting the results so set the top of the screen red
+                alert("Error getting rankings. Are you sure you're connected to the field network?");
```
Comment (fangeugene) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: This alert isn't firing for me.

Comment (phil-lopreiato) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: Check the network inspector in devtools and see if the request is timing out?

Comment (fangeugene) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: Actually, no request is being made.
`Uncaught TypeError: Cannot read property 'value' of undefined`

Comment (phil-lopreiato) [inline static/javascript/tba_js/eventwizard_apiwrite.js]: I'll fix it all up tonight when I get home from work.

**Pull request merged**